run-sonar-scanner.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -eo pipefail
+
+if [[ "$RUNNER_OS" == "Windows" ]]; then
+  SCANNER_BIN="sonar-scanner.bat"
+else
+  SCANNER_BIN="sonar-scanner"
+fi
+
+scanner_args=()
+if [[ ${SONARCLOUD_URL} ]]; then
```

Comment:
I have added this to support the future forward of the SC GitHub action (`sonarcloud-github-action`) to this action, see [here](https://github.com/SonarSource/sonarcloud-github-action/compare/master...antonioaversa:sonarcloud-github-action:master).